lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -111,6 +111,8 @@ def __init__(self):
         """
         Initialize the s1aplibrary and its callbacks.
         """
+        self._imsi_idx = 1
+        self.IMSI_LEN = 15
```

Comment:
will you be adding a test case in this PR? maybe move to the PR that has the actual test cases?